ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/node-type-l3-interfaces.yml

```diff
@@ -47,7 +47,6 @@ ethernet_interfaces:
   ip_address: 192.168.1.2/31
   shutdown: false
   type: routed
-  description: peer1_eth1
```

Comment:
This should not be removed. It is probably caused by the changes to the custom modules. Once they are reverted, this should come back.